### DISCUSSION TITLE
Aqara roller shade companion E1

### DISCUSSION THREAD

**Discussion by skaker1** [Koenkk/zigbee2mqtt#10310] [Devices]
I can't get the battery level for this cover:

Aqara roller shade companion E1 (ZNJLBL01LM)

I have tried to reset it and repaired it. I also have tried a new one that I bought with the same problem.

Is there a fix?

**Comment by fewdji** [comment]
Also didn't get it. Got error in Log:
`ERROR (MainThread) [homeassistant.helpers.template] Error parsing value: access to attribute 'update' of 'dict' object is unsafe. (value: {"battery":null,"linkquality":69,"position":0,"running":false,"update_available":null}, template: {{ value_json['update']['state'] == "available" }})`

**Comment by Koenkk** [comment]
Duplicate of https://github.com/Koenkk/zigbee2mqtt/issues/9447#issuecomment-968709485
